### DISCUSSION TITLE
How to remove Sidebar Title Bar?

### DISCUSSION THREAD

**Discussion by SHHSSH** [piroor/treestyletab#3654] [Q&A] [answered]
Hi, in recent Beta/Developer versions Firefox has changed something I believe that has had my userchrome settings or TST not function correctly and not hiding the sidebar title bar, does anyone know how to resolve this?

Thanks.

![image](https://github.com/user-attachments/assets/eda0954d-0e4d-4bea-b753-c11cd48b86bb)

**Comment by irvinm** [comment]
It would be a userChrome.css thing.

What CSS are you currently using that was working until recently for this behavior?  What version of FF specifically isn't working now?

**Reply by SHHSSH** [reply]
> It would be a userChrome.css thing.
> 
> What CSS are you currently using that was working until recently for this behavior? What version of FF specifically isn't working now?

I'm going to copy and paste my whole userChrome.css, it's crazy messy down the bottom half of it due to me manually removing certain context menu items etc.

133.0b4 (64-bit) Developer Edition

` #titlebar {
  visibility: collapse;
}

#main-window[inFullscreen] #sidebar-box {
  display:none !important;
  width: 0px !important;
}

#PlacesToolbarItems toolbarseparator { opacity: 0 !important; }

/* remove maximum/minimum width restriction of sidebar */
#sidebar-box {
  max-width: none !important;
  min-width: 11px !important;
}

#main-window #TabsToolbar {
    opacity: 0;
    max-height: 0px;
    pointer-events: none;
}

#navigator-toolbox {
  background-color: rgb(236, 233, 216) !important;
  background-image: linear-gradient(to top, rgba(10, 31, 51, 0.35) 2px, rgba(0, 0, 0, 0.05) 2px, transparent 50%) !important;
}

/* Use Normal top and bottom padding for Compact */
#PlacesToolbarItems .bookmark-item {
  padding-top: 11px !important;
  padding-bottom: 1px !important;
}	

#tabbar tab-closebox {
	display: none; 
}

.menubar-text,
#toolbar-menubar .toolbarbutton-text,
.tabbrowser-tab .tab-label {
  color: black !important;
}

/*
   Override hiding of Bookmarks Toolbar in F11 full screen
   when you roll down the tabs/main toolbar 
*/
#navigator-toolbox[inFullscreen="true"] #PersonalToolbar {
  visibility: unset !important;
}

#main-menubar menu:hover .menubar-text {
  color: white !important;
}

#context-navigation, #context-sendpagetodevice { 
    display: none !important 
}

#context-navigation, #context-sendlinktodevice { 
    display: none !important 
}

#context-navigation, #_73a6fe31-595d-460b-a920-fcc0f8843232_-menuitem-_noscript-ctx-menu { 
    display: none !important 
}

#context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_selectAllTabs { 
    display: none !important 
}

;     adblockultimate@adblockultimate.net

#context-navigation, #adblockultimate_adblockultimate_net-menuitem- { 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-""{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-_{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-_Adblocker Ultimate{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-_Adblocker Ultimate{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-31{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-32{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-33{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-34{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-35{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-36{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-37{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-38{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-39{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-40{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-41{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-42{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-43{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-44{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-45{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-46{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-47{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-48{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-49{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-50{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-51{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-52{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-53{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-54{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-55{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-56{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-57{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-58{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-59{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-60{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-30{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-29{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-28{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-27{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-26{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-25{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-24{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-23{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-22{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-21{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-20{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-19{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-18{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-17{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-16{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-15{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-14{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-13{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-12{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-11{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-10{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-9{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-8{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-7{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-6{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-5{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-4{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-3{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-2{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-1{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-61{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-62{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-63{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-64{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-65{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-66{
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-67{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-68{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-69{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-70{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-71{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-72{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-73{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-74{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-75{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-76{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-77{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-78{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-79{ 
    display: none !important
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-80{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-81{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-82{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-83{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-84{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-85{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-86{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-87{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-88{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-89{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-90{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-91{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-92{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-93{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-94{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-95{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-96{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-97{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-98{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-99{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-100{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-101{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-102{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-103{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-104{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-105{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-106{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-107{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-108
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-109{
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-110{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-111{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-112{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-113{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-114{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-115{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-116{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-117{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-118{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-119{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-120{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-121{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-123{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-124{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-125{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-126{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-127{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-128{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-129{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-130{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-131{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-132{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-133{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-134{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-135{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-136{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-5644{ 
    display: none !important 
}

#context-navigation, #adblockultimate_adblockultimate_net-menuitem-{ 
    display: none !important 
}

context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-6 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-7 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-8 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-9 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-10 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-11{ 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-12 { 
    display: none !important 
}


context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-13 { 
    display: none !important 
}

context-navigation, #_c2c003ee-bd69-42a2-b0e9-6f34222cb046_-menuitem-1 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-1 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-2 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-3 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-4 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-5 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-6 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-7 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-8 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-9 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-10 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-11 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-12 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-13 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-14 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-15 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-16 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-19 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-17 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-18 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-20 { 
    display: none !important 
}

context-navigation, #_531906d3-e22f-4a6c-a102-8057b88a1a63_-menuitem-3587 { 
    display: none !important 
}

context-navigation, #adblockultimate_adblockultimate_net-menuitem-1402 { 
    display: none !important 
}

context-navigation, #ublock0_raymondhill_net-menuitem-1397 { 
    display: none !important 
}

context-navigation, #ublock0_raymondhill_net-menuitem-_uBlock0-blockElement { 
    display: none !important 
}

context-navigation, #context-savepage { 
    display: none !important 
}

context-navigation, #context-pocket { 
    display: none !important 
}

context-navigation, #context-selectall { 
    display: none !important 
}

context-navigation, #context-take-screenshot { 
    display: none !important 
}

context-navigation, #context-inspect-a11y { 
    display: none !important 
}

context-navigation, #_windscribeff-menuitem-_view-debug-log { 
    display: none !important 
}

}


#sidebar {
  max-width: 1200px !important;
}

#context-navigation, #_73a6fe31-595d-460b-a920-fcc0f8843232_-menuitem-_noscript-ctx-menu { 
    display: none !important 
}

#context-navigation, #jid0-bnmfwww2w2w4e4edvcddbnmhdvg_jetpack-menuitem-1 { 
    display: none !important 
}

#context-navigation, #context-openlinkinusercontext-menu {
	display: none !important;
}

#context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_reopenInContainer { 
    display: none !important 
}

#context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_selectAllTabs { 
    display: none !important 
}

; context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_closeTab { 
;     display: none !important 
; }

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_closeMultipleTabs { 
    display: none !important 
}

; context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_duplicateTab { 
;    display: none !important 
; }

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_reloadTab { 
    display: none !important 
}

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_topLevel_reloadTree { 
    display: none !important 
}

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_toggleMuteTab-mute { 
    display: none !important 
}

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_sendTabsToDevice { 
    display: none !important 
}

! context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_pinTab { 
!     display: none !important 
! }

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_newTab { 
    display: none !important 
}

! context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_moveTab { 
! display: none !important 
! }

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_bookmarkTab { 
    display: none !important 
}

context-navigation, #treestyletab_piro_sakura_ne_jp-menuitem-_context_undoCloseTab { 
    display: none !important 
}

#contentAreaContextMenu menuseparator {
    display: none !important;
}

tab-item:not(.pinned) {
  padding-left: 0px !important; /* !important is required when there are enough tabs to cause a scrollbar */
}`

**Reply by irvinm** [reply]
You have some items in here to hide the entire sidebar when in fullscreen, but I don't see anything for "sidebar-header" so I am not sure how this ever worked.

Basically if you never want to see the sidebar header, just add "display: none" to that element.

```
#sidebar-header {
	display: none;
}
```

Before:
![image](https://github.com/user-attachments/assets/4c61df25-c330-4923-9fff-8deb6ead293f)

After:
![image](https://github.com/user-attachments/assets/c6dc2143-cdc8-49aa-b2b2-41c9f0804bab)

**Reply by SHHSSH** [reply]
@irvinm Sweet, thanks, that worked.

I reckon I had been removing it via about:config setting's in the past until recent UI changes.

**Reply by irvinm** [reply, marked as answer]
You are welcome!  Can you mark this issue as "answered"?  Thanks.

**Comment by SHHSSH** [comment]
I searched for "sidebar" but I didn't think the stuff was relevant

**Comment by piroor** [comment]
By the way, the sidebar header goes away on the new style sidebar on Nightly.
![GcJpnwgawAEL_NX](https://github.com/user-attachments/assets/5f9a3e1c-38d8-4454-977e-bee78cf6ccd0)
Instead the thin bar replacing the sidebar header takes space. It looks just a needless space, but it become the native vertical tabs with configured.
![GcJor6lasAEacmz](https://github.com/user-attachments/assets/9c314570-5f39-447a-ab15-78486abb646f)
![GcJnU4sasAQXpVa](https://github.com/user-attachments/assets/7e6341d3-690c-4708-83a6-e70647e625ff)
